pyproject.toml

```diff
@@ -1,15 +1,19 @@
 [build-system]
-requires = [
-    "setuptools >= 43.0.0",
-    "wheel >= 0.32.0",
-]
+requires = ["setuptools>=60", "setuptools_scm[toml]>=7.0"]
```

Comment:
For reproducibility, I think it's better for this section to pin exact version numbers, as I did in Toga.